libs/ui/src/lib/membership-card/membership-card.component.scss

```diff
@@ -7,6 +7,14 @@
   padding-top: calc(1 * var(--borderWidth));
   width: 100%;
 
+  .card-wrapper {
```

Comment:
Could you move all definitions (e.g. `.card-container`) into `.card wrapper { ... }`?